logbook-core/src/test/java/org/zalando/logbook/CommonsLogFormatSinkTest.java

```diff
@@ -26,6 +28,11 @@
 
     private final HttpLogWriter writer = mock(HttpLogWriter.class);
 
+    @BeforeEach
+    void setUp() {
+        Locale.setDefault(Locale.ENGLISH);
```

Comment:
Do you happen to have the error (message) at hand that you experienced without this?

Reply:
![image](https://user-images.githubusercontent.com/21218309/114193010-c833b080-9956-11eb-8fd6-9895fe7593c8.png)
As you can see, month letters are cyrillic on my machine with the default locale (Russian in my case) and it causes test failures.
And I decided that changing formatter locale in `CommonsLogFormatSink` would be breaking change so I added this setup.

Reply:
Actually, I believe you discovered a bug. CLF is using US date time format, afaik.

Reply:
Well, I supposed the same way as you. But specification says [here](https://en.wikipedia.org/wiki/Common_Log_Format) and [here](https://strftime.org/):
```
[10/Oct/2000:13:55:36 -0700] is the date, time, and time zone that the request was received, by default in strftime format %d/%b/%Y:%H:%M:%S %z.
.....
%b - Month as **locale’s** abbreviated name (i.e. Sep)
```